src/main/java/uk/gov/pay/api/agreement/model/CreateAgreementRequest.java

```diff
@@ -12,41 +12,66 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateAgreementRequest {
     
-    public static final int REFERENCE_MIN_LENGTH = 1;
-    public static final int REFERENCE_MAX_LENGTH = 255;
+    public static final String USER_IDENTIFIER_FIELD = "user_identifier";
+
+    public static final int MIN_LENGTH = 1;
+    public static final int MAX_LENGTH = 255;
 
     @JsonProperty("reference")
-    @Size(min= REFERENCE_MIN_LENGTH, max = REFERENCE_MAX_LENGTH, message = "Must be less than or equal to {max} characters length")
+    @Size(min= MIN_LENGTH, max = MAX_LENGTH, message = "Must be less than or equal to {max} characters length")
     private String reference;
 
+    @JsonProperty("description")
+    @Size(min= MIN_LENGTH, max = MAX_LENGTH, message = "Must be less than or equal to {max} characters length")
+    private String description;
+
+    @JsonProperty(USER_IDENTIFIER_FIELD)
+    @Size(min= MIN_LENGTH, max = MAX_LENGTH, message = "Must be greater than or equal to {min} characters length and less than or equal to {max} characters length")
```

Comment:
It looks like similar fields use the message “Must be less than or equal to 255 characters length”, which we should probably match for consistency.

Reply:
I agree it’s weird but I think consistency (even if it’s consistently weird!) is desirable here.